### PR TITLE
Add pay pal confirm logic

### PR DESCRIPTION
**Pull request by neunygph**
When checking out with PayPal, there was no confirmation step to take the user back to check out pay to confirm the payment. Adding this step will enable users the option to confirm their purchase.

### CODE REVIEW

File: theme/src/components/checkoutForm/paymentForm/PayPalCheckout.js [hunk 1]
```diff
@@ -32,8 +32,7 @@ export default class PayPalButton extends React.Component {
 			{
 				// Set your environment
 				env: formSettings.env, // sandbox | production
-				// Show the buyer a 'Pay Now' button in the checkout flow
-				commit: true,
+
 				// Specify the style of the button
 				style: {
```
Comment (LuBuss) [inline theme/src/components/checkoutForm/paymentForm/PayPalCheckout.js:37]: Are we able to apply the "Pay with PayPal" button styling to the "Confirm Payment" button

**Review by LuBuss — APPROVED** [review]
Tested and working.
Confirm button does not apply the same styling and Pay Button.

File: theme/src/components/checkoutForm/paymentForm/PayPalCheckout.js (hunk 1 above)
Comment (neunygph) [reply theme/src/components/checkoutForm/paymentForm/PayPalCheckout.js:37]: We can with custom css, but I’m not a designer so I don’t know how :)

Comment (LuBuss) [reply theme/src/components/checkoutForm/paymentForm/PayPalCheckout.js:37]: Let me see what I can do today to get them both the buttons in sync.

**Comment by neunygph** [comment]
Thanks

**Pull request merged**